### DISCUSSION TITLE
A shortcut to run it with any config

### DISCUSSION THREAD

**Discussion by c0mrad12** [hacksider/Deep-Live-Cam#489] [Show and tell]
just make a shortcut and make it something like C:\Windows\System32\cmd.exe /k  cd C:\Windows\System32\Deep-Live-Cam & python run.py --execution-provider openvino
so first cd into deep live cam and then do a & symbol and enter the code u would need to run it , it makes it alot more convenient but it isnt really that smart

**Comment by hacksider** [comment]
There's a .bat there where you can modify or just create a shortcut on the desktop. 😅
